config/installation_config.yml

```
@@ -42,6 +42,10 @@
   value: 'Chatwoot'
   display_title: 'Brand Name'
   description: 'The name that would be used in emails and the widget'
+- name: BRAND_COLOR
+  value: '#1f93ff'
+  display_title: 'Brand Color'
```

YAMLlint (1.38.0)

[error] string value is redundantly quoted with any quotes

(quoted-strings)